frontend/app/views/tier/upgrade/paidToPaid.scala.html

```diff
@@ -0,0 +1,148 @@
+@(
+    currentTier: com.gu.membership.salesforce.Tier,
+    targetTier: com.gu.membership.salesforce.Tier,
+    userFields: model.PrivateFields,
+    pageInfo: model.PageInfo,
+    paidPreviewOpt: Option[model.Zuora.PaidPreview],
+    currentSubscription: model.Zuora.SubscriptionDetails
+)(implicit token: play.filters.csrf.CSRF.Token)
+
+@import helper._
+@import org.joda.time.DateTime
+@import views.support.Prices._
+@import views.support.Dates._
+@import views.support.Asset
+
+@main("Upgrade to " + targetTier, pageInfo=pageInfo) {
+
+    <main role="main" class="page-content l-constrained">
+
+        @fragments.page.pageHeader("Great, glad to see you've decided to become a " + targetTier)
+
+        <div class="page-section page-section--bordered">
+            <div class="page-section__lead-in">
+                <h2 class="page-section__headline">What happens now</h2>
+            </div>
+            <div class="page-section__content">
+                <p>When you upgrade we want to make sure you are charged the right amount. We will charge for your <strong>@currentTier</strong> membership up until today and the amount remaining will be deducted from you're first payment as a <strong>@targetTier</strong>.</p>
+                <p><strong>@targetTier</strong> payments start today and will re-occur @{currentSubscription.paymentPeriodLabel}ly.</p>
```

Comment:
More language pedantry but "re-occur" should be "recur" (cf. [Grammarist](http://grammarist.com/usage/recur-reoccur/)).